pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -999,9 +1008,17 @@ customization:
           resource: clusterroles
           subresource: ""
           name: ""
+    pinnedResources:
+    - group: apps
+      version: v1
+      resource: Deployment
+    - group: core
```

Comment:
"core" or ""? I'm fine with both, but we should align this then with the enhancement doc example.